src/api/cvc4cpp.cpp

```diff
@@ -2124,6 +2141,61 @@ Term Solver::mkBitVector(const std::string& s, uint32_t base) const
   return mkBVFromStrHelper(s, base);
 }
 
+Term Solver::mkBitVector(uint32_t size, const char* s, uint32_t base) const
+{
+  std::string str(s);
+  return mkBitVector(size, str, base);
+}
+
+Term Solver::mkBitVector(uint32_t size, std::string& s, uint32_t base) const
+{
+  Integer val(s, base);
+  CVC4_API_CHECK(val.modByPow2(size) == val)
+      << "Overflow in bitvector construction (specified bitvector size " << size
+      << " too small to hold value " << s << ")";
+  return d_exprMgr->mkConst(BitVector(size, val));
```

Comment:
Use`mkConstHelper()` (wrapper that ensures type checking, throws CVC4APIException).

Reply:
Done (the method now calls the new `mkBVFromStrHelper()` which uses `mkConstHelper()`.